ocs_ci/utility/rosa.py

```diff
@@ -71,20 +71,38 @@ def create_cluster(cluster_name, version, region):
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
     rosa_mode = config.ENV_DATA.get("rosa_mode", "")
+    private_link = config.ENV_DATA.get("private_link", "")
```

Comment:
Same as above, `ENV_DATA["private_link"]` is used as boolean, so I would suggest to not set string as default value:

```suggestion
    private_link = config.ENV_DATA.get("private_link")
```

Reply:
Updated as  `private_link = config.ENV_DATA.get("private_link", False)`